v2_bridges/erc20_on_int/src/int/metadata.rs

```diff
@@ -10,20 +10,29 @@ impl Erc20OnIntEthTxInfo {
     fn to_metadata(&self) -> Result<Metadata> {
         let user_data = if self.user_data.len() > MAX_BYTES_FOR_ETH_USER_DATA {
             info!(
-                "✘ `user_data` redacted from `Metadata` ∵ it's > {} bytes",
+                "`user_data` redacted from `Metadata` ∵ it's > {} bytes",
```

Comment:
:+1: I've basically been removing these non-ascii symbols from any comments in code I'm touching in various PRs, glad you're doing the same. 

I liked them in the early days since they stood out nicely in logs etc, but then they turned out to cause issues in some places where the symbol doesn't exist etc. Lesson learned!

Reply:
Yeah I was adhering closely to your own PR, I figured it might be related to that!